kartothek/io/testing/read.py

```diff
@@ -366,6 +366,40 @@ def test_read_dataset_as_dataframes_concat_primary(
     pdt.assert_frame_equal(expected_df, result_df, check_like=True)
 
 
+def test_read_dataset_as_dataframes_dispatch_by(
+    store_factory, custom_read_parameters, bound_load_dataframes, output_type
+):
+    if output_type != "dataframe":
```

Comment:
I believe we do not need to skip the dataframe backend here

Reply:
That too bad, my tests currently fail for `output_type == "metapartition"` or `output_type == "table"`. Will have to look into it.